lib/Controller/DocumentController.php

```diff
@@ -1131,6 +1142,16 @@ private function putRelative($fileId, $owner, $editor, $suggested) {
 			$root->newFolder(\dirname($path));
 		}
 
+		try {
+			$view = new View();
```

Comment:
any alternative on creating the view instance here?
In addition, creating the view for the root folder might be confusing if the path used in the `verifyPath` function isn't relative to the root folder. Someone could try a `$view->getFileInfo($path)` later, that could fail because the path refers to the user's root folder, not to the root folder.
Furthermore, it makes more sense that the `verifyPath` checks the information based on the specific view. I mean, a path might be valid for the root folder, but not for other folders. I don't think we have use cases like this, but it feels more natural.